### PR TITLE
Add Travis and update README

### PR DESCRIPTION
**Pull request by chaithyagr**
This resolves the issue I raised in #23 
I have added the travis,yml, updated the unittests to work and Pass.
Locally I have a passing build.

### CODE REVIEW

**Comment by chaithyagr** [comment]
Adding @LElgueddari  and @zaccharieramzi  for review.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+sudo: required
```
Comment (LElgueddari) [inline .travis.yml]: Could you shorten the file so It will no have a redundant installation with respect to pysap?

Comment (chaithyagr) [reply .travis.yml]: I have shortened it now and pushed locally. 
For now, we can just wait for it to pass. However, if things get too delayed and messy, I suggest we push this long version for now and later work on shortening.
In my opinion getting a Travis script up is at the highest priority. We can track the shortening in issue #24

Comment (chaithyagr) [reply .travis.yml]: Done @LElgueddari , I have shortened it to minimum code in my opnion

**Comment by zaccharieramzi** [comment]
I'd rather have @AGrigis review this since I am not familiar with travis. Regarding the tests part, lgtm, but it's not the main story here.

**Comment by chaithyagr** [comment]
Add @AGrigis for review and final merge at this point.

**Comment by AGrigis** [comment]
The travis file seems good.
I think you can merge.

**Comment by chaithyagr** [comment]
Thanks @AGrigis 
Merging and will close #23

**Pull request merged**